spec/classes/puppet_agent_windows_install_spec.rb

```diff
@@ -52,6 +52,7 @@
             it { is_expected.to contain_file('C:\tmp\install_puppet.bat').with_content(
                 %r[#{Regexp.escape("msiexec.exe /qn /norestart /i \"#{values[:appdata]}\\Puppetlabs\\packages\\puppet-agent-#{values[:expect_arch]}.msi\"")}])
             }
+            it { is_expected.to contain_file('C:\tmp\install_puppet.bat').with_content(/PUPPET_AGENT_ENVIRONMENT="rp_env"/) }
```

Comment:
It is the value of the $::environment fact while running in rspec-puppet... interestingly I have yet to figure out how to override that specific fact/top scope variable in rspec... but for this use case it works to test absence of typo/quoting issues...